index.html

```diff
@@ -11,32 +11,201 @@
 			body {
 				font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Helvetica Neue", Arial, sans-serif;
 			}
+
+			#search-field-label {
+				display: block;
+			}
+
+			.container {
+				max-width: 1028px;
+				margin: 0 auto;
+			}
+
+			.page-title, .search-form {
+				text-align: center;
+			}
+
+			.article-image {
+				height: 127px;
+				width: 190px;
+				flex: 1;
+			}
+
+			.text-container {
+				flex: 3;
+				margin-left: 1em;
+			}
+
+			.result-item {
+				display: flex;
+				margin: 2em;
+				padding: 1em;
+				border: 2px solid rgba(0, 0, 0, .5);
+			}
+
+			#search-field, #search-button {
+				height: 2em;
+				font-size: 18px;
+			}
+
+			#search-button {
+				margin-left: 1em;
+				display: inline-block;
+			}
+
 		</style>
 	</head>
 
 	<body>
-		<h1>Thrilling Frontend Technical Challenge</h1>
+		<div class="container">
+			<h1 class="page-title">Thrilling Frontend Technical Challenge</h1>
+
+			<form class="search-form">
+				<input type="text" placeholder="Enter Search Term" id="search-field" />
+				<input type="button" value="Search" id="search-button" />
+			</form>
 
-		<h2>Open your Developer Tools Console to view output.</h2>
+			<div id="results-list"></div>
+		</div>
 
 		<script>
-			var request = new XMLHttpRequest();
 
-			request.open('GET', 'https://api.nytimes.com/svc/topstories/v2/science.json?api-key=Gwxln5M3geWlhR6UE0TY1FUWKSG3wCil');
+			const resultsList = document.querySelector('#results-list');
+			const searchField = document.querySelector('#search-field');
+			const searchButton = document.querySelector('#search-button');
+
+			searchField.addEventListener('keydown', (event) => {
+				if (event.keyCode === 13) {
+					event.preventDefault();
+					const request = setupRequest();
+					request.send();
+				}
+			});
+
+			searchButton.addEventListener('click', (event) => {
+				event.preventDefault();
+
+				const request = setupRequest();
+				request.send();
+			});
+
+
+			function setupRequest () {
+				const request = new XMLHttpRequest();
+
+				request.open('GET', 'https://api.nytimes.com/svc/topstories/v2/science.json?api-key=Gwxln5M3geWlhR6UE0TY1FUWKSG3wCil');
+
+				request.onload = handleResponse;
+
+				request.onerror = function() {
+					console.error(this);
+				};
+
+				return request;
+			}
+
+			function handleResponse () {
 
-			request.onload = function() {
 				if (this.status >= 200 && this.status < 400) {
-					console.log(JSON.parse(this.response));
+					const parsedResponse = JSON.parse(this.response);
+					if (parsedResponse.results && Array.isArray(parsedResponse.results)) {
+
+						// remove existing children from results
+						while (resultsList.firstChild) {
+							resultsList.removeChild(resultsList.firstChild);
+						}
+
+						// split search input into array so order doesn't matter
+						const searchInput = searchField.value.split(' ');
```

Comment:
```suggestion
						const searchInput = searchField.value.trim().split(/\s+/);
```
Consider as a way to make the conditional block from 129-144 unnecessary, as well as prevent more cases where blank spaces could be inadvertently entered and searched.

Reply:
This is really cool, that makes sense